06-job-control.md

```diff
@@ -4,8 +4,169 @@ title: Extra Unix Shell Material
 subtitle: Job control
 minutes: 5
 ---
+
 > ## Learning Objectives {.objectives}
 >
 > * FIX ME
 
-FIX ME
\ No newline at end of file
+Our next topic is how to control programs *once they're running*. This
+is called [job control](glossary.html#job-control), and while it's less
+important today than it was back in the Dark Ages, it is coming back
+into its own as more people begin to leverage the power of computer
+networks.
```

Comment:
That last half sentence is rather vague and isn't adding much. What has this to do with networks?